src/components/events/view-events/StatusButton.test.js

```diff
@@ -18,17 +18,17 @@ describe("Test StatusButton static properties", () => {
       <Provider store={store}>
         <BrowserRouter>
           <StatusButton
-            name={"Going"}
+            name={"Maybe"}
             color={"#58D573"}
             eventId={1}
-            eventStatus={"Maybe"}
+            eventStatus={"Going"}
```

Comment:
Should this say "Maybe", @kyle-richardson?

Reply:
future tests should be created for more functional unit testing, but for the test purposes currently in the file, it is referencing the "Maybe" name which tells the component which button it is rendering, thus the check for text on line 32 relies on line 21 only.  line 24 is separate, which references the current status on the event.  in the case for this test, it is an unnecessary prop